Src/BlazorApp/Pages/OngoingRaces.razor

```diff
@@ -71,10 +73,10 @@
 
 @code
 {
-    private List<Race> _races = new List<Race>();
```

Comment:
```csharp 
private List<Race> _races = new List<Race>();
```
In C# you don't need to say what type a variable is 2 times.

Reply:
...Since I have declared the variabel in Race.cshtml?

Reply:
No.

Reply:
No.
@terjetopland 
`int myNumber = 42`
Here is it stated 2 times that myNumber is int. 42 is int and int is int. DRY. So here you can write instead:
`var myNumber = 42`.

Instead repeating List<Race> 2 times 

private List<Race> _races = new List<Race>();

You can write:
private List<Race> Races = new();